lib/models/building.dart

```diff
@@ -1,14 +1,11 @@
-import 'package:flutter/material.dart';
+import 'package:concordia_navigation/storage/app_constants.dart';
```

Comment:
Consider adding a synonym to make it clear in code that a constant is from that file, i.e., `import 'package:concordia_navigation/storage/app_constants.dart' as constants;`